### DISCUSSION TITLE
URDF Model Appears Flipped in Foxglove Studio with ROS 2

### DISCUSSION THREAD

**Discussion by 3bdul1ah** [foxglove/community#1065] [Q&A]
### Discussed in https://github.com/orgs/foxglove/discussions/1063

<div type='discussions-op-text'>

<sup>Originally posted by **3bdul1ah** December 29, 2024</sup>
I'm experiencing an issue with my robot visualization:

1. The robot model displays correctly in RViz2
2. However, when viewing in Foxglove Studio, the same model appears flipped
3. The URDF and robot_description topic are working properly

![image](https://github.com/user-attachments/assets/5b940b55-d3aa-4137-aa4f-a95a1bd1ff7f)

Has anyone encountered this behaviour? What might be causing this difference between RViz2 and Foxglove?

</div>

**Comment by acarrillo-civ** [comment]
In case anyone else comes to this page, the solution is here: https://github.com/orgs/foxglove/discussions/1063

Scene --> Mesh-up "Z" --> CTRL-R Refresh
